### PR TITLE
add fragment orders and order dishes to activity

### PR DESCRIPTION
**Pull request by lamvuvan**

### CODE REVIEW

File: app/src/main/java/com/framgia/awesomecity/screen/orders/MyPagerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        mContext = context.getApplicationContext();
+    }
+
+    @Override
+    public Fragment getItem(int position) {
+        switch (TABS[position]){
+            case ORDERS:
+                return OrdersFragment.newInstance(mContext.getString(R.string.title_orders));
+            case ORDER_DISHES:
+                return OrderDishesFragment.newInstance(mContext.getString(R.string.title_order_dishes));
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/MyPagerAdapter.java]: ERROR: Line is longer than 100 characters (found 104).

File: app/src/main/java/com/framgia/awesomecity/screen/orders/MyPagerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        switch (TABS[position]){
+            case ORDERS:
+                return mContext.getString(R.string.title_orders);
+            case ORDER_DISHES:
+                return mContext.getString(R.string.title_order_dishes);
+        }
+        return null;
+    }
+
+    public void addFragment(Fragment fragment, String title){
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/screen/orders/MyPagerAdapter.java]: ERROR: '{' is not preceded with whitespace.

File: app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java [hunk 3]
```diff
@@ -15,4 +20,16 @@ public static void setAdapter(RecyclerView recyclerView, RecyclerView.Adapter ad
         recyclerView.setAdapter(adapter);
         recyclerView.addItemDecoration(new DividerItemDecoration(recyclerView.getContext(), DividerItemDecoration.VERTICAL));
     }
+
+    @BindingAdapter("pagerAdapter")
+    public static void setViewPager(ViewPager viewPager, OrdersActivity activity){
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java]: ERROR: '{' is not preceded with whitespace.

File: app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java [hunk 4]
```diff
@@ -15,4 +20,16 @@ public static void setAdapter(RecyclerView recyclerView, RecyclerView.Adapter ad
...
+        MyPagerAdapter adapter = new MyPagerAdapter(activity.getSupportFragmentManager(), viewPager.getContext());
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java]: ERROR: Line is longer than 100 characters (found 114).

File: app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java [hunk 5]
```diff
@@ -15,4 +20,16 @@ public static void setAdapter(RecyclerView recyclerView, RecyclerView.Adapter ad
...
+        viewPager.setAdapter(adapter);
+    }
+
+    @BindingAdapter("viewPager")
+    public static void setupWithViewPager(TabLayout tabLayout, ViewPager viewPager){
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/awesomecity/utils/BindingUtils.java]: ERROR: '{' is not preceded with whitespace.

**Comment by lamvuvan** [comment]
move to ubuntu

**Pull request closed without merging**